entrypoint.sh

```diff
@@ -4,8 +4,17 @@ set -e
 
 readonly keyboard="$1"
 readonly keymap="$2"
-readonly qmk_output="$3"
-readonly local_keymap="$4"
+readonly controller="$3"
+readonly qmk_output="$4"
+readonly local_keymap="$5"
+
+# Check if the controller matches one of the available options
+if [ "$controller" ]; then
+  if [ "$controller" != "proton_c" ] && [ "$controller" != "kb2040" ] && [ "$controller" != "promicro_rp2040" ] && [ "$controller" != "blok" ] && [ "$controller" != "bit_c_pro" ] && [ "$controller" != "stemcell" ] && [ "$controller" != "bonsai_c4" ] && [ "$controller" != "rp2040_ce" ] && [ "$controller" != "elite_pi" ] && [ "$controller" != "helios" ] && [ "$controller" != "liatris" ] && [ "$controller" != "imera" ] && [ "$controller" != "michi" ]; then
+    echo "Invalid convert option $keyboard"
+    exit 1
+  fi
+fi
```

Comment:
I believe we should let the QMK CLI perform this check instead. More controllers might come out and become supported by QMK, and when that happens, I'd rather not have to sync the list of valid strings here with the supported controllers.

What do you think of dropping this section altogether?